DIY Reminders with Azure and Twilio/blog.md

```diff
@@ -1,9 +1,9 @@
 ## Azure Alarms
-​	I'm a very forgetful person, especially when I'm out having fun. I love hiking and biking, but my allergies make life miserable so I need to take my meds every 24 hours when I head out. To fix this problem, I can just set a notification, but when I'm doing something else, I can very easily miss the slight buzz and tone from my phone.
+​	I'm a very forgetful person, especially when I'm out having fun. I love hiking and biking, but my allergies make life miserable so I need to take my meds every 24 hours when I head out. To fix this problem, I can just set a notification, but when I'm doing something else, it's easy to miss the slight buzz and tone from my phone.
 
-​	With Azure and Twilio, this problem can be easily remedied. We'll use Azure to set up a timer to trigger other code to run, and integrate Twilio so we can send a text message or call to ourselves to make sure we notice the reminder.
+​	With Azure and Twilio, this problem can be simply remedied. We'll use Azure to set up a timer to trigger other code to run and integrate Twilio, so we can send a text message or call to ourselves to make sure we notice the reminder.
 
-​	It will take a little bit of Javascript to make everything work, but it's pretty straightforward and we'll go over everything together. Firstly, we need to set up 2 accounts, Microsoft Azure , and Twilio. Azure has a 12 month free trial and Twilio has a free trial as well with some credits to experiment with the features.
+​	It will take a little bit of Javascript to make everything work, but it's pretty straightforward and we'll go over everything together. Firstly, we need to set up 2 accounts, [Microsoft Azure](https://azure.microsoft.com/en-us/free/search/?&ef_id=Cj0KCQjwn7j2BRDrARIsAHJkxmwz0tcFx8fTgoL7jogctsTPYrzEntPYcIpEu0QV_YRDbPFyv7oNrOoaAvspEALw_wcB:G:s&OCID=AID2000128_SEM_Cj0KCQjwn7j2BRDrARIsAHJkxmwz0tcFx8fTgoL7jogctsTPYrzEntPYcIpEu0QV_YRDbPFyv7oNrOoaAvspEALw_wcB:G:s&gclid=Cj0KCQjwn7j2BRDrARIsAHJkxmwz0tcFx8fTgoL7jogctsTPYrzEntPYcIpEu0QV_YRDbPFyv7oNrOoaAvspEALw_wcB) and [Twilio](https://www.twilio.com/try-twilio). Azure has a 12 month free trial, and Twilio has a free trial as well with some credits to experiment with the features.
 
 ### Step 1: Create Timer Function
 ​	Let's pop open Azure and click to create a new resource, get started, then select "Function App". After that, fill in all the info about your function. Make sure you select "JavaScript" for the runtime stack option.
```

Comment:
These instructions here were a bit confusing. Do I need to click on "To Create a New Resource" in Azure to get started?